### ISSUE TITLE
Muilti-monitor and/or top edge scrolling support

### ISSUE THREAD

**Issue by SiebrenW** [squgeim/Workspace-Scroll#4]
Hi, I just installed this extension as a replacement for the Desktop Scroller (https://extensions.gnome.org/extension/561/desktop-scroller-all-sides-for-gnome-36/) extension, and I quite like the fact I have desktop scrolling again.
However I use multiple monitors and full screen VMs mostly, which DS could deal with nicely, just move the cursor to the top edge (or whichever you configured to use) and scroll, and it would work on both my extra monitor and with a fullscreen VM open.

Is a similar feature an option for this extension? 

TL;DR: So mainly: I want to scroll workspaces with a fullscreen window open and on another monitor.

**Comment by squgeim** [comment]
I will definitely look into implementing something like it but I can't guarantee a deadline. Also, pull requests are always welcome. 🙂
